### ISSUE TITLE
Snapshot: Missing language in output

### ISSUE THREAD

**Issue by IchHabRecht** [fastlane/fastlane#10787]
### New Issue Checklist

- [x] Updated fastlane to the latest version
- [x] I read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I read [docs.fastlane.tools](https://docs.fastlane.tools)
- [x] I searched for [existing GitHub issues](https://github.com/fastlane/fastlane/issues)

### Issue Description

I'm missing the information which language is currently be processed.

I'm running snapshot with following configuration

```
+----------------------------+--------------------------------------+
|                    Summary for snapshot 2.63.0                    |
+----------------------------+--------------------------------------+
| devices                    | ["iPhone 7 Plus", "iPhone 8 Plus"]   |
| ios_version                | 11.0.1                               |
| languages                  | ["en-US", "de-DE"]                   |
| scheme                     | BirdeesSnapshotUITests               |
| output_directory           | /Users/nicole/Developer/Birdees/scr  |
|                            | eenshots                             |
| clear_previous_screenshots | true                                 |
| project                    | ./Birdees.xcodeproj                  |
| launch_arguments           | ["without-animations"]               |
| clean                      | true                                 |
| concurrent_simulators      | false                                |
| stop_after_first_error     | true                                 |
| output_simulator_logs      | false                                |
| skip_open_summary          | false                                |
| skip_helper_version_check  | false                                |
| reinstall_app              | false                                |
| erase_simulator            | false                                |
| localize_simulator         | false                                |
| buildlog_path              | ~/Library/Logs/snapshot              |
| number_of_retries          | 3                                    |
| xcode_path                 | /Applications/Xcode.app              |
+----------------------------+--------------------------------------+
```

I know that first "en-US" is processed, afterwards "de-DE". The output only says, which device is processed but not with which language:

```
[18:36:34]: Running snapshot on: iPhone 7 Plus
[18:55:14]: Running snapshot on: iPhone 8 Plus
[19:03:21]: Running snapshot on: iPhone 7 Plus
[19:13:53]: Running snapshot on: iPhone 8 Plus
```

**Comment by IchHabRecht** [comment]
I'm still unable to see the current processed language as there is no information in the logs.

**Comment by KrauseFx** [comment]
Is this something that has worked in the past?

**Comment by IchHabRecht** [comment]
Yes, with a version for Xcode 8 I saw the current used language in the logs.

**Comment by KrauseFx** [comment]
Ahhh, I think that's because with Xcode 9 we now run multiple simulators in parallel, meaning that it probably runs tests for languages for that specific device. Any specific reason you need to know the language it's running right now?

**Comment by IchHabRecht** [comment]
But are you really running multiple simulators with different languages in parallel? I have the feeling that each language is processed separately - thus there shouldn't be any problem showing the current language. For me this is important to get the status of current tests and get an overview what is processed already.

**Comment by ohayon** [comment]
Hey @IchHabRecht - Yeah you're right we are only running parallel simulators with the same language at once. We should be able to print out which language is being run at a given time. That being said, I believe higher up in the output, it will say that it is running tests for each language individually. Could you share the full output so we can see exactly whats going on? Thanks! 🚀

**Comment by IchHabRecht** [comment]
Hi @ohayon which output do you mean?

**Comment by IchHabRecht** [comment]
I finally managed to test the latest stable version. The problem with seeing the current processed language is still valid.

**Comment by IchHabRecht** [comment]
Still an open issue
